README.md

```diff
@@ -1,17 +1,141 @@
-# Project
+# GitHub Action for deploying to Azure Spring Cloud
+GitHub Actions support an automated software development lifecycle workflow. With GitHub Actions for Azure Spring Cloud you can create workflows in your repository to manage your deployment of Azure Spring Cloud conveniently.
 
-> This repo has been populated by an initial template to help get you started. Please
-> make sure to update the content to build a great experience for community-building.
+[comment]: <> (This repository contains GitHub Action for Azure Spring Cloud to deploy. The action supports deploying *.jar, *.war, and *.zip files, or a folder.)
 
-As the maintainer of this project, please make a few updates:
+[comment]: <> (As the maintainer of this project, please make a few updates:)
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
+[comment]: <> (- Improving this README.MD file to provide a great experience)
 
-## Contributing
+[comment]: <> (- Updating SUPPORT.MD with content about this project's support experience)
+
+[comment]: <> (- Understanding the security reporting process in SECURITY.MD)
+
+[comment]: <> (- Remove this section from the README)
+## Prerequisites
+### Set up GitHub repository and authenticate
+You need an Azure service principal credential to authorize Azure login action. To get an Azure credential, execute the following commands on your local machine:
```

Comment:
We can delegate this to the online doc: https://docs.microsoft.com/en-us/cli/azure/create-an-azure-service-principal-azure-cli . It's better to let the customer follow the official doc to operate on the service principals. I think we don't need to repeat the steps here.

Reply:
Sure, good idea